specs/_features/epbs/beacon-chain.md

```diff
@@ -0,0 +1,731 @@
+    message: ExecutionPayloadHeader
+    signature: BLSSignature
+```
+    
+#### `ExecutionPayloadEnvelope`
+
+```python
+class ExecutionPayloadEnvelope(Container):
+    payload: ExecutionPayload
+    builder_index: ValidatorIndex
+    beacon_block_root: Root
+    blob_kzg_commitments: List[KZGCommitment, MAX_BLOB_COMMITMENTS_PER_BLOCK]
+    inclusion_list_proposer_index: ValidatorIndex
+    inclusion_list_slot: Slot
+    inclusion_list_signature: BLSSignature
+    payload_withheld: bool
+    state_root: Root
+```
+
+#### `SignedExecutionPayloadEnvelope`
+
+```python
+class SignedExecutionPayloadEnvelope(Container):
+    message: ExecutionPayloadEnvelope
+    signature: BLSSignature
+```
+
+#### `InclusionListSummary`
+
+```python
+class InclusionListSummary(Container)
+    proposer_index: ValidatorIndex
+    slot: Slot
+    summary: List[ExecutionAddress, MAX_TRANSACTIONS_PER_INCLUSION_LIST]
+```
+
+#### `SignedInclusionListSummary`
+
+```python
+class SignedInclusionListSummary(Container):
+    message: InclusionListSummary
+    signature: BLSSignature
+```
+
+#### `InclusionList`
+
+```python
+class InclusionList(Container)
+    signed_summary: SignedInclusionListSummary
+    parent_block_hash: Hash32
+    transactions: List[Transaction, MAX_TRANSACTIONS_PER_INCLUSION_LIST]
+```
+
+### Modified containers
+
+
+#### `BeaconBlockBody` 
+**Note:** The Beacon Block body is modified to contain a Signed `ExecutionPayloadHeader`. The containers `BeaconBlock` and `SignedBeaconBlock` are modified indirectly.
+
+```python
+class BeaconBlockBody(Container):
+    randao_reveal: BLSSignature
+    eth1_data: Eth1Data  # Eth1 data vote
+    graffiti: Bytes32  # Arbitrary data
+    # Operations
+    proposer_slashings: List[ProposerSlashing, MAX_PROPOSER_SLASHINGS]
+    attester_slashings: List[AttesterSlashing, MAX_ATTESTER_SLASHINGS]
+    attestations: List[Attestation, MAX_ATTESTATIONS]
+    deposits: List[Deposit, MAX_DEPOSITS]
+    voluntary_exits: List[SignedVoluntaryExit, MAX_VOLUNTARY_EXITS]
+    sync_aggregate: SyncAggregate
+    # Execution
+    # Removed execution_payload [Removed in ePBS]
+    # Removed blob_kzg_commitments [Removed in ePBS]
+    bls_to_execution_changes: List[SignedBLSToExecutionChange, MAX_BLS_TO_EXECUTION_CHANGES]
+    # PBS
+    signed_execution_payload_header: SignedExecutionPayloadHeader  # [New in ePBS]
+    payload_attestations: List[PayloadAttestation, MAX_PAYLOAD_ATTESTATIONS] # [New in ePBS]
+```
+
+#### `ExecutionPayload`
+
+**Note:** The `ExecutionPayload` is modified to contain a transaction inclusion list summary signed by the corresponding beacon block proposer.
+
+```python
+class ExecutionPayload(Container):
+    # Execution block header fields
+    parent_hash: Hash32
+    fee_recipient: ExecutionAddress  # 'beneficiary' in the yellow paper
+    state_root: Bytes32
+    receipts_root: Bytes32
+    logs_bloom: ByteVector[BYTES_PER_LOGS_BLOOM]
+    prev_randao: Bytes32  # 'difficulty' in the yellow paper
```

Comment:
```suggestion
    prev_randao: Bytes32  # 'mix_hash' in the yellow paper
```

Reply:
has this changed in the YP and the main repo? I took this one out from the main repo